### PR TITLE
Updating README.md based on siddhi site change

### PR DESCRIPTION
**Pull request by HamsaV**
To resolve this issue
https://github.com/wso2-extensions/siddhi-io-mqtt/issues/5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,65 @@
...
+
+The **siddhi-io-mqtt** is an extension to <a target="_blank" href="https://wso2.github.io/siddhi">Siddhi</a> mqtt source and sink implementation,that publish and receive events from mqtt broker.
+
+Find some useful links below:
+
+* <a target="_blank" href="https://github.com/wso2-extensions/siddhi-io-mqtt">Source code</a>
+* <a target="_blank" href="https://github.com/wso2-extensions/siddhi-io-mqtt/releases">Releases</a>
+* <a target="_blank" href="https://github.com/wso2-extensions/siddhi-io-mqtt/issues">Issue tracker</a>
+## Latest API Docs 
+
```
Comment (tishan89) [inline README.md]: Latest API doc should come here. Ask Chathurika.

File: README.md [hunk 2]
```diff
@@ -1 +1,65 @@
...
+## How to use 
+
+**Using the extension in <a target="_blank" href="https://github.com/wso2/product-sp">WSO2 Stream Processor</a>**
+
+* You can use this extension in the latest <a target="_blank" href="https://github.com/wso2/product-sp/releases">WSO2 Stream Processor</a> that is a part of <a target="_blank" href="http://wso2.com/analytics?utm_source=gitanalytics&utm_campaign=gitanalytics_Jul17">WSO2 Analytics</a> offering, with editor, debugger and simulation support. 
+
+* This extension is shipped by default with WSO2 Stream Processor, if you wish to use an alternative version of this extension you can replace the component <a target="_blank" href="https://github.com/wso2-extensions/siddhi-execution-unique/releases">jar</a> that can be found in the `<STREAM_PROCESSOR_HOME>/lib` directory.
```
Comment (tishan89) [inline README.md]: releases link should be changed

File: README.md [hunk 3]
```diff
@@ -1 +1,65 @@
...
+
+---
+
+## Features
+
+
+## How to Contribute
+ 
+  * Please report issues at <a target="_blank" href="https://github.com/wso2-extensions/siddhi-io-mqtt/issues">GitHub Issue Tracker</a>.
+  * Send your bug fixes pull requests to <a target="_blank" href="https://github.com/wso2-extensions/siddhi-io-mqtt/tree/master">master branch</a>.
```
Comment (tishan89) [inline README.md]: Make the sentence as uniform as unique extension

File: README.md [hunk 4]
```diff
@@ -1 +1,65 @@
...
+ 
+## Contact us 
+
+ * Post your questions on <a target="_blank" href="http://stackoverflow.com/">Stackoverflow</a> with the tag <a target="_blank" href="http://stackoverflow.com/search?q=siddhi">"siddhi"</a>.
```
Comment (tishan89) [inline README.md]: Make sentence as uniform as unique extesnion

File: README.md (hunk 1 above)
Comment (HamsaV) [reply README.md]: fixed in 3ba15c9

File: README.md (hunk 2 above)
Comment (HamsaV) [reply README.md]: fixed in 3ba15c9

File: README.md (hunk 4 above)
Comment (HamsaV) [reply README.md]: fixed in 3ba15c9

File: README.md [hunk 5]
```diff
@@ -1 +1,71 @@
...
+---
+
+## Features
+
+* <a target="_blank" href="https://wso2-extensions.github.io/siddhi-io-mqtt/api/1.0.2-SNAPSHOT/#mqtt-sink">mqtt</a> *(<a target="_blank" href="https://wso2.github.io/siddhi/documentation/siddhi-4.0/#sinks">Sink</a>)*<br><div style="padding-left: 1em;"><p>The MQTT sink publishes the events to an MQTT broker </p></div>
+* <a target="_blank" href="https://wso2-extensions.github.io/siddhi-io-mqtt/api/1.0.2-SNAPSHOT/#mqtt-source">mqtt</a> *(<a target="_blank" href="https://wso2.github.io/siddhi/documentation/siddhi-4.0/#sources">Source</a>)*<br><div style="padding-left: 1em;"><p>The MQTT source receives the events from an MQTT broker </p></div>
+
+## How to Contribute
+ 
+   * Please report issues at <a target="_blank" href="https://github.com/wso2-extensions/siddhi-execution-unique/issues">GitHub Issue Tracker</a>.
```
Comment (tishan89) [inline README.md]: What I meant when saying to make this as same as unique is to make the sentence composition the same. Your hyperlinks should still point to mqtt repo

File: README.md [hunk 6]
```diff
@@ -1 +1,71 @@
...
+
+   * Send your contributions as pull requests to <a target="_blank" href="https://github.com/wso2-extensions/siddhi-execution-unique/tree/master">master branch</a>
```
Comment (tishan89) [inline README.md]: same as above

File: README.md (hunk 5 above)
Comment (HamsaV) [reply README.md]: fixed in be22da7

File: README.md (hunk 6 above)
Comment (HamsaV) [reply README.md]: fixed in be22da7

**Pull request merged**